src/app/organizations/collection/collection.component.html

```diff
@@ -101,18 +104,28 @@ <h3 *ngIf="Dockstore.FEATURES.enableNotebooks">Workflows, Notebooks, and Tools</
             <h3 *ngIf="!Dockstore.FEATURES.enableNotebooks">Workflows and Tools</h3>
 
             <div data-cy="collectionEntry" *ngFor="let entry of collection?.entries">
-              <mat-card class="pb-3">
+              <mat-card
+                *ngIf="
+                  (entry.entryType !== entryType.Notebook && !Dockstore.FEATURES.enableNotebooks) || Dockstore.FEATURES.enableNotebooks
+                "
+                class="pb-3"
+              >
                 <a
                   [routerLink]="[
-                    '/' + (entry.entryType === 'tool' ? 'container' : 'workflow') + 's/',
+                    '/' +
+                      (entry.entryType === entryType.Tool
+                        ? 'container'
+                        : entry.entryType === entryType.Workflow
```

Comment:
IDE error on `entryType.Workflow` -- it doesn't exist.

Reply:
Oh woah thanks for catching this, supposed to be `BioWorkflow`.  What IDE do you use Charles? That's really useful

Reply:
I use WebStorm, but if you're using IntelliJ, I think you can configure it with the same plugins as WebStorm.